taskana-adapter-camunda-outbox-rest/src/main/java/pro/taskana/adapter/camunda/outbox/rest/resource/CamundaTaskEventResourcesWrapper.java

```diff
@@ -0,0 +1,27 @@
+package pro.taskana.adapter.camunda.outbox.rest.resource;
+
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * This class wraps a list of CamundaTaskEventResources in a seperate object.
+ * This is needed to make the REST-API work with Jersey
+ *
+ *
+ */
+
+public class CamundaTaskEventResourcesWrapper implements Serializable {
```

Comment:
In our other rest services we have a naming convention, which would lead to "CamundaTaskEventListResource". Is there a reason,, why you don't comply to that convention? Furthermore we use Assembler to create the resources.

Compare this to the WorkbasketController 👍 
        WorkbasketSummaryListResource pagedResources = workbasketSummaryResourceAssembler.toResources(
            workbasketSummaries,
            pageMetadata);

        ResponseEntity<WorkbasketSummaryListResource> response = ResponseEntity.ok(pagedResources);

Reply:
Added model classes + renamed the Wrapper to fulfill Resource naming convention. Added Resource Assembler